docs/identity/authentication/howto-authentication-passwordless-security-key-on-premises.md

```diff
@@ -140,7 +140,7 @@ _(Example: For US Government Cloud)_
    $cloudCred = Get-Credential -Message 'An Active Directory user who is a member of the Hybrid Identity Administrators group for Microsoft Entra ID.'
 
    # Enter a Domain Administrator username and password.
-   $domainCred = Get-Credential -Message 'An Active Directory user who is a member of the Domain Admins group.'
+   $domainCred = Get-Credential -Message 'An Active Directory user who is a member of the Domain Admins group And an Enterprise Admin for the forest.'
```

Comment:
```suggestion
   $domainCred = Get-Credential -Message 'An Active Directory user who is a member of the Domain Admins group and an Enterprise Admin for the forest.'
```
@Justinha Let me know if I understood this correctly to make this change, or if it should be discarded. Thanks!

Reply:
Nudging @Justinha

Reply:
@v-dirichards I'm sorry to miss this! Yes please make this change and thank you!